src/app/space/create/deployments/apps/deployment-details.component.ts

```diff
@@ -27,6 +27,8 @@ import 'patternfly/dist/js/patternfly-settings.js';
 })
 export class DeploymentDetailsComponent {
 
+  readonly DEFAULT_SPARKLINE_DATA_DURATION: number = 15 * 60 * 1000;
```

Comment:
Changing this to `public static readonly` seems to work for me. Can you try again?

Reply:
To add to this: rather than using the customizer to set `@Input()` fields, these should be set through the template similar to how it's done in an actual use scenario.

See `deployments-donut-chart.component.spec.ts` for an example